build.gradle

```diff
@@ -12,14 +12,16 @@ buildscript {
     }
 
     dependencies {
+        classpath 'com.netflix.nebula:gradle-info-plugin:5.0.3'
+        classpath 'com.netflix.nebula:gradle-info-plugin:5.0.3'
```

Comment:
wat

Reply:
Looks like there's an issue with the excavator